pkgs/applications/audio/master_me/default.nix

```
@@ -0,0 +1,40 @@
+{ lib, stdenv, fetchFromGitHub, libGL, libX11, libXext, libXrandr, libXinerama, libXcursor, freetype, alsa-lib, cmake, pkg-config, gcc-unwrapped, python3 }:
```

```suggestion
{ lib, stdenv, fetchFromGitHub, libGL, libX11, libXext, libXrandr, libXinerama, libXcursor, freetype, alsa-lib, cmake, pkg-config, python3 }:
```
Please double check for other unused inputs

---

I'm working on it, but I ran a nix profile command, and now ``nix-env -f $NIXPKGS -iA master_me`` doesn't work anymore: ``error: profile '/nix/var/nix/profiles/per-user/bart/profile' is incompatible with 'nix-env'; please use 'nix profile' instead``.

---

Done.